Automattic/newspack-listings
# chore(deps): update newspack-scripts

### All Submissions:






### Changes proposed in this Pull Request:

Similar to https://github.com/Automattic/newspack-theme/pull/1845 and https://github.com/Automattic/newspack-blocks/pull/1174, updates prettier, stylelint and Newspack scripts to the latest versions:

* newspack-scripts@4.0.0 was installed
* wp-prettier (aliased to prettier) were updated
* styles were migrated with `npx sass-migrator module src/**/*.scss`
* styles were fixed with `npm run lint:scss -- --fix`
* JS was fixed with `npm run lint:js -- --fix`

I also took a pass diffing the generated CSS and JS files before and after this change; the main difference I noticed is that this PR generates assets.** files that are not included in builds prior to this update (assets.asset.php, assets.css, assets.js, assets.rtl.css). 

I don't think this causes any issues exactly, but please let me know if it seems concerning! 

### How to test the changes in this Pull Request:

1. Review the automated changes
2. Confirm all jobs pass in CI
3. Confirm `npm run build` runs without error. 

### Other information:
